app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+    validates :first_name, presence: true, length:{maximum: 30},format: { with: /\A[a-zA-Z]+\z/}
```

Comment:
Great, this is how to test using a regex 👍

Reply:
thank you